assets/src/hooks/useVehicles.ts

```diff
@@ -229,10 +237,12 @@ const vehicleFromData = (vehicleData: VehicleData): Vehicle => ({
   scheduledHeadwaySecs: vehicleData.scheduled_headway_secs,
   dataDiscrepancies: dataDiscrepanciesFromData(vehicleData.data_discrepancies),
   stopStatus: vehicleStopStatusFromData(vehicleData.stop_status),
-  timepointStatus: vehicleTimepointStatusFromData(vehicleData.timepoint_status),
-  scheduledTimepointStatus: vehicleTimepointStatusFromData(
-    vehicleData.scheduled_timepoint_status
-  ),
+  timepointStatus:
+    vehicleData.timepoint_status &&
```

Comment:
If only we had matching in JS…

Reply:
😢I miss case expressions and the [`Maybe` type](https://package.elm-lang.org/packages/elm/core/1.0.2/Maybe#map)